webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -224,15 +239,42 @@ export const GlideTable: React.FC<Props> = ({
     [data, columnIds, columnDefs],
   );
 
-  const handleGridSelectionChange = useCallback((newSelection: GridSelection) => {
-    const [, row] = newSelection.current?.cell ?? [undefined, undefined];
+  const handleCellClicked = useCallback((cell: Item) => {
+    const [, row] = cell;
```

Comment:
What is this change for? 

I think this may get in the way of implementing shift+clicking for ranges.

Reply:
in my PR this changed selecting a row from any-click to left-click.  Right-clicking to open a context menu had been simultaneously adding that row to the selection

I see what you're saying about multi-select or keyboard nav features on the table
In that case maybe `handleGridSelectionChange` could handle selecting multiple rows and `handleCellClicked` could handle selecting single rows?

Reply:
ah okay, yeah i think we can work that out later then.